README.md

```diff
@@ -1,16 +1,20 @@
-# PyVer (WIP)
+# PyVer
 
-[![License](https://img.shields.io/badge/license-MIT-blue?style=flat-square)](LICENSE-MIT)
+<!-- markdownlint-disable MD013 -->
+![GitHub](https://img.shields.io/github/license/Allstreamer/pyver) [![🧪 Tests](https://github.com/Allstreamer/pyver/actions/workflows/tests.yml/badge.svg)](https://github.com/Allstreamer/pyver/actions/workflows/tests.yml) [![🖋  Check linting](https://github.com/Allstreamer/pyver/actions/workflows/lint.yml/badge.svg)](https://github.com/Allstreamer/pyver/actions/workflows/lint.yml) [![🔨 Build](https://github.com/Allstreamer/pyver/actions/workflows/build.yml/badge.svg)](https://github.com/Allstreamer/pyver/actions/workflows/build.yml) [![📦 Package](https://github.com/Allstreamer/pyver/actions/workflows/package.yml/badge.svg)](https://github.com/Allstreamer/pyver/actions/workflows/package.yml) [![📄 Build docs](https://github.com/Allstreamer/pyver/actions/workflows/docs.yml/badge.svg)](https://github.com/Allstreamer/pyver/actions/workflows/docs.yml) [![👔 Check formatting](https://github.com/Allstreamer/pyver/actions/workflows/format.yml/badge.svg)](https://github.com/Allstreamer/pyver/actions/workflows/format.yml)
+<!-- markdownlint-enable MD013 -->
 
-> **Python PEP-440 Version Parsing**
+> **Python PEP-440 Version Parser**
 
-This package allows for parsing Python PEP-440 version numbers and comparisons between PEP-440 Versions
+This package allows for parsing Python [PEP-440](https://peps.python.org/pep-0440/)
+version numbers and for comparisons between
+[PEP-440](https://peps.python.org/pep-0440/) version numbers.
 
 ## Usage
 
 ```Toml
 [dependencies]
-pyver = "0.0.1"
+pyver = "1"
 ```
```

Comment:
Did you add more examples to the README? Maybe just add a few more

Reply:
There are examples most things the the rust docs

Reply:
Then you can just add like 2 or 3 more and tell them to look into the cargo docs for more

Reply:
@John15321 2 examples have been added + links to crates.io and docs.rs